.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+- With issues:
+  - Use the search tool before opening a new issue.
+  - Please provide source code and commit sha if you found a bug.
+  - Review existing issues and provide feedback or react to them.
+
+## Description
```

Comment:
Need templates for:
+ Report a bug
+ Request a feature
+ Ask a question

Reply:
done

Reply:
moved to `.github` repo